src/ngraph/runtime/cpu/unit_test.manifest

```diff
@@ -24,3 +24,6 @@ lrn_6D_across_2_axes
 
 # ONNX TopK with dynamic K
 top_k_opset_10
+
+# ONNX TopK with dynamic K
+top_k_opset_10
```

Comment:
There is duplicate `top_k_opset_10`. Didn't you intended to write: `top_k_opset_11_const_k_smallest` ?